app/models/appointment.rb

```diff
@@ -36,6 +36,13 @@ def self.overdue
     where(status: 'scheduled').where('scheduled_date <= ?', Date.today)
   end
 
+  def self.appointments_per_facility
```

Comment:
I'd call this `group_by_facility`, as it's already in the `Appointment` class

Reply:
Ha, yeah. Just an artifact from before. Still a work in progress. I will change this name, it makes sense.